### PR TITLE
Module 3: Bookstore - Initialize Project with Components

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 3 (Week 2): Bookstore - Initialize Project with Components
For this project, I
- Set up a new react app using create react app.
- Added directory for your reusable `components`.
- Implemented two routes using `react-router-dom` one for the `Homepage` and the `Categories` page.
- Added component for the `book`, as well as a button to delete the book.
- Added a `book list` component that displays individual books.
- Implemented a `form` to create new books.
- Included `navigation links` for the two routes.

### CODE REVIEW

File: src/components/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default function BooksList() {
+  return (
+    <div>
+      <h2>No books added yet</h2>
+    </div>
+  );
```
Comment (brytebee) [inline src/components/BooksList.js]: Books are required to be rendered on the page. Kindly add a book structure here and attempt to render them to the page.

File: src/components/AddNewBook.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default function AddNewBook() {
+  return (
+    <section>
+      <h2>ADD NEW BOOK</h2>
+      <form>
+        <input type="text" name="book-title" placeholder="Book title" />
+        <input type="text" name="author" placeholder="Author" />
+      </form>
```
Comment (brytebee) [inline src/components/AddNewBook.js]: Great work adding a form. 👏👏👏

However, the form isn't controlled. There is no action added to handle the interaction from users.
To do this, you are required to add a submit button which at the moment might not function.

*Kindly add a button to this form.*

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ikennarichard 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @ikennarichard,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**